src/protocol/modulus_conversion/convert_shares.rs

```diff
@@ -182,9 +199,8 @@ mod tests {
         let match_key = MaskedMatchKey::mask(rng.gen());
         let result: [Replicated<Fp31>; 3] = world
             .semi_honest(match_key, |ctx, mk_share| async move {
-                convert_one_bit(ctx, RecordId::from(0), &mk_share, BITNUM)
-                    .await
-                    .unwrap()
+                let triple = convert_bit_local::<Fp31>(ctx.role(), BITNUM, &mk_share);
```

Comment:
This is good. Is it worth adding another test for Fp32BitPrime?

Reply:
Incoming with the malicious tests.